src/battle/battle_icon.c

```diff
@@ -9,13 +9,12 @@
 #include "struct_defs/sprite_template.h"
 #include "struct_defs/struct_0200D0F4.h"
 
-#include "unk_0200C6E4.h"
-
 #include "narc.h"
+#include "unk_0200C6E4.h"
 
 /*
-* Maps a type index (e.g. TYPE_WATER) to the corresponding NARC member in pl_batt_obj.narc containing the type icon.
-*/
+ * Maps a type index (e.g. TYPE_WATER) to the corresponding NARC member in pl_batt_obj.narc containing the type icon.
+ */
 __attribute__((aligned(4))) static const u32 BattleIcon_NARCMemberMap[] = {
     0xEA,
```

Comment:
**polish**: If you generate a `naix` file using `knarc`, then you can include it in this file like any other C header and get named constants for these, which will be better for conceptual links throughout the project.

I would also suggest leveraging array-initializer syntax here to self-document these tables, e.g.,:

```c
static const u32 sBattleTypeIcon_NARCMemberMap[] = {
    [TYPE_NORMAL] = ...,
    [TYPE_FIGHTING] = ...,
    [TYPE_FLYING] = ...,
};
```